### PR TITLE
Added ReturnTypeDeclarationFixer

### PR DESCRIPTION
**Pull request by keradus**
Support for `Return Type Declarations` (part of #1119)

### CODE REVIEW

**Comment by keradus** [comment]
cc @gharlan

**Comment by GrahamCampbell** [comment]
Maybe add a couple of tests with newlines and multiple comments in too?

**Comment by keradus** [comment]
could you send PR to mine one ?

File: src/Fixer/FunctionNotation/ReturnTypeDeclarationFixer.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class ReturnTypeDeclarationFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return PHP_VERSION_ID >= 70000 && $tokens->isTokenKindFound(T_FUNCTION);
```
Comment (gharlan) [inline src/Fixer/FunctionNotation/ReturnTypeDeclarationFixer.php]: why not checking `CT_TYPE_COLON`?

**Pull request closed without merging**